js/src/core.ts

```diff
@@ -275,7 +275,8 @@ export function getEncodingNameForModel(model: TiktokenModel) {
       return "cl100k_base";
     }
     case "gpt-4o":
-    case "gpt-4o-2024-05-13": {
+    case "gpt-4o-2024-05-13":
+    case "gpt-4o-mini": {
```

Comment:
We should also add the name of the model version, as is done for other models.
```suggestion
    case "gpt-4o-2024-05-13":
    case "gpt-4o-mini-2024-07-18":
    case "gpt-4o-mini": {
```